tests/backends/test_pyethereum21.py

```diff
@@ -23,6 +24,8 @@
 def eth_tester():
     if not is_pyethereum21_available():
         pytest.skip("PyEthereum >=2.0.0,<2.2.0 not available")
+    if os.environ.get('ETHEREUM_TESTER_VALIDATOR') is not None:
+        del os.environ['ETHEREUM_TESTER_VALIDATOR']
```

Comment:
you should use the [`monkeypatch`](https://docs.pytest.org/en/latest/monkeypatch.html) fixture to accomplish this.